src/text-area/index.tsx

```diff
@@ -169,13 +170,18 @@ export const TextArea = factory(function TextArea({
 		onValidate
 	} = properties();
 
-	let value = icache.get('value');
-	const existingInitialValue = icache.get('initialValue');
+	let { value } = properties();
+	const hasValue = typeof value !== 'undefined';
```

Comment:
I don't think this will ever not be true due to the default in the destructuring.
How about using the same approach I did in text-input?

https://github.com/dojo/widgets/pull/1357/files#diff-be70a03acb1add410d9f0b3bd7817872R172-R184

Reply:
That default in the destructuring is in a different function, btw, the diff is just trying to deceive you.